lib/api/admin_api.ts

```diff
@@ -417,6 +417,12 @@ export const specs = {
       properties: {
         email: { $ref: "#/definitions/EmailAddress" },
         preferred_languages: { $ref: "#/definitions/PreferredLanguages" },
+        is_storage_of_message_content_enabled: {
```

Comment:
`$ref` here too?

Reply:
About using $ref or embedding the definition, actually there's no clear rule. Sometimes we use $ref(s) some others we embed the definitions (ie. CreatedMessage -> sender_service_id).

Reply:
when in doubt I would prefer avoiding duplication of definitions as they may go out of sync